src/wing/CCPACSWing.cpp

```diff
@@ -1454,6 +1459,21 @@ void CCPACSWing::CreateNewConnectedElementAfter(std::string startElementUID)
             if ( elementsBefore.size() < 2) {
                 throw  CTiglError("Impossible to add a element after if there is no previous element");
             }
+
+            // Iterate over segments to find the one ending in startElementUID
+            // If the corresponding segment contains guide curves -> Throw error, since adding elements after gc-segments is not supported
+            for (int i=1; i <= GetSegmentCount(); i++)
+            {
+                if (GetSegment(i).GetOuterSectionElementUID() == startElementUID)
+                {
+                    if(GetSegment(i).GetGuideCurves())
+                    {
+                        throw tigl::CTiglError("Adding sections after wing segments containing guide curves is currently not supported.\n"
+                                               "In general, guide curves should only be added when all sections are already defined, since the guide curves depend on them.", TIGL_UID_ERROR);
+                    }
+                }
```

Comment:
See above.

```suggestion
                if(GetSegment(i).GetGuideCurves())
                {
                    throw tigl::CTiglError("Adding sections after wing segments containing guide curves is currently not supported.\n"
                                           "In general, guide curves should only be added when all sections are already defined, since the guide curves depend on them.", TIGL_UID_ERROR);
               }
```